cmd/earthly/main.go

```diff
@@ -792,7 +792,21 @@ func (app *earthlyApp) before(context *cli.Context) error {
 	return nil
 }
 
+func (app *earthlyApp) warnIfEarth() {
+	if len(os.Args) == 0 {
+		return
+	}
+	binPath := os.Args[0] // can't use os.Executable() here; because it will give us earthly if executed via the earth symlink
+
+	baseName := path.Base(binPath)
+	if baseName == "earth" {
+		app.console.Warnf("Warning: the earth binary has been renamed to earthly; the earth command is currently symlinked, but is deprecated and will one day be removed.")
```

Comment:
We should also say... `Feel free to rm <path-to-earth>`, if we detect that `earthly` exists.

Reply:
that's a good idea, I just pushed up a new version to do that.

Reply:
Forgot to add files maybe?

Reply:
I don't think so, I did a rebase+force push.

Reply:
Oh I see https://github.com/earthly/earthly/pull/637

Reply:
nvm, I got wires crossed :-)